<?xml version="1.0" encoding="utf-8"?>
<report
	progress="">
	<activity
		result="Failed"
		duration="21.9m"
		durationms="1312823"
		type="root"
		rid="16f0b86a42bdf21"
		totalerrorcount="1"
		totalwarningcount="0"
		totalsuccesstestcasecount="0"
		totalfailedtestcasecount="0"
		totalblockedtestcasecount="0"
		childsuccesscount="0"
		childfailedcount="0"
		childblockedcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		totalmaintenancemodecount="0"
		user="Fahad Shah"
		rxversion="8.0.0.0"
		host="HUAWEI LUA-U22 (Unknown)
WLAN - 192.168.3.11"
		osversion="-"
		runtimeversion="-"
		procarch="-"
		oslanguage="-"
		dtnlanguage="-"
		screenresolution="-"
		timestamp="2/23/2018 12:10:24 PM"
		timestampiso="2018-02-23T07:10:24"
		endtime="2/23/2018 12:32:17 PM"
		timeoutfactor="1"
		module="VisitOtherShopOptions">
		<detail>
			VisitOtherShopOptions
		</detail>
		<varbindings>
		</varbindings>
		<item
			timeRelativeToTestSuiteStartTime="21:52.823"
			timeRelativeToTestModuleStartTime="21:52.823"
			timeWallClock="12:32:17"
			time="21:52.823"
			level="Error"
			category="Module">
			<message>
				The test run has been aborted by the user.
			</message>
			<metainfo
				loglvl="Error">
			</metainfo>
		</item>
	</activity>
</report>
